File: SEM_5/Python/Assignment-4/Q1.py
import random
n = int(input("Enter the size of the list: "))
l = [random.randint(1, 100) for _ in range(n)]
avg = sum(l) / n if n > 0 else 0
print("Random List:", l)
print("Sum:", sum(l))
print("Average:", avg)

File: SEM_1 (ICP)/Array/Q1.java
import java.util.Scanner;
public class Q1{
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter any value n: ");
        int n=sc.nextInt();
        double sum=0;
        int a[]=new int[n];
        sc.close();
        for (int i=0;i<n; i++ ){
            a[i]=1+(int)(Math.random()*n);
        }
        for(int i=1; i<=n; i++){
            sum+=a[i];
        }
        double avg=sum/n;
        System.out.print(sum+"\n"+avg);
    }
}
